src/vs/workbench/parts/tasks/electron-browser/task.contribution.ts

```diff
@@ -610,6 +610,9 @@ class TaskService extends EventEmitter implements ITaskService {
 
 	private registerCommands(): void {
 		CommandsRegistry.registerCommand('workbench.action.tasks.runTask', (accessor, arg) => {
```

Comment:
Maybe there's somewhere downstream to put this so that it gets hit no matter how I run a task - cmd+shift+b, or "Run test task" or